ci/mullvad-browser/download-mullvad-browser.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 # TODO: Uncomment when alpha is to be released
```

Comment:
As the MB alpha can be added to the repositories at the earliest opportunity, we can remove this line.

Reply:
Good catch! Clumsy of me to not notice. Fixed